app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None | str:
+    file_names = command.split()
+
+    if len(file_names) != 3 or file_names[1] == file_names[2]:
+        return "invalid format"
+
+    old_file, new_file = file_names[1], file_names[2]
+
+    # Copy the file content from source to destination
```

Comment:
You can remove this comment, because it is the same as your function name, the logic is clear